macros/models/_sector/tokens/transfers_base.sql

```diff
@@ -0,0 +1,65 @@
+{% macro transfers_base(blockchain, traces, transactions, erc20_transfers) %}
+{%- set token_standard_20 = 'bep20' if blockchain == 'bnb' else 'erc20' -%}
+{# denormalized tables are not yet in use #}
+{%- set denormalized = True if blockchain in ['base'] else False -%}
+
+WITH transfers AS (
+    SELECT block_time
+    , block_number
+    , tx_hash
+    , value AS amount_raw
+    , CAST(NULL AS varbinary) AS contract_address
+    , 'native' AS token_standard
+    , "from"
+    , to
+    , NULL AS evt_index
+    , trace_address
+    FROM {{ traces }}
+    WHERE success
+    AND (call_type NOT IN ('delegatecall', 'callcode', 'staticcall') OR call_type IS null)
+    AND value > UINT256 '0'
+    {% if is_incremental() %}
+    AND {{incremental_predicate('block_time')}}
+    {% endif %}
+
+    UNION ALL
+
+    SELECT t.evt_block_time AS block_time
+    , t.evt_block_number AS block_number
+    , t.evt_tx_hash AS tx_hash
+    , t.value AS amount_raw
+    , t.contract_address
+    , '{{token_standard_20}}' AS token_standard
+    , t."from"
+    , t.to
+    , t.evt_index
+    , NULL AS trace_address
+    FROM {{ erc20_transfers }} t
```

Comment:
if we filter native transfers with `value > 0`, should we do the same for erc20?

Reply:
I checked and there are about 25M rows that have `value = 0` (can't be negative due to the type). No idea why this happens but here's a few example events https://etherscan.io/tx/0x347f678c54e205ed54d0b7c7a77f19e8447e239ce4908123eec6f7059c6b2f4e#eventlog

Reply:
Yes it's due to devs not checking their logic before calling a `transferFrom()`